volttrontesting/fixtures/volttron_platform_fixtures.py

```diff
@@ -194,9 +197,7 @@ def volttron_instance(request, **kwargs):
                             **kwargs)
 
     def cleanup():
-        print('Shutting down instance: {}'.format(wrapper.volttron_home))
-        wrapper.remove_all_agents()
-        wrapper.shutdown_platform()
+        cleanup_wrapper(wrapper)
```

Comment:
Can you add cleanup_wrapper() in volttron_instance_rmq() and volttron_instance_zmq() fixtures